shared/src/ledger/eth_bridge/bridge_pool_vp.rs

```diff
@@ -77,6 +80,58 @@ where
             Some(SignedAmount::Positive(after - before))
         }
     }
+
+    /// Check that the correct amount of Nam was sent
+    /// from the correct account into escrow
+    fn check_nam_escrowed(
+        &self,
+        payer_account: &Address,
+        escrow_account: &Address,
+        expected_debit: Amount,
+        expected_credit: Amount,
+    ) -> bool {
+        let debited = self.account_balance_delta(payer_account);
+        let credited = self.account_balance_delta(escrow_account);
+        if debited.is_none() && credited.is_none() {
+            return false;
+        }
+
+        match (debited, credited) {
+            (
+                Some(SignedAmount::Negative(debit)),
+                Some(SignedAmount::Positive(credit)),
+            ) => debit == expected_debit && credit == expected_credit,
+            (Some(SignedAmount::Positive(_)), _) => {
+                tracing::debug!(
+                    "The account {} was not debited.",
+                    payer_account
+                );
+                false
+            }
+            (_, Some(SignedAmount::Negative(_))) => {
+                tracing::debug!(
+                    "The Ethereum bridge pool's escrow was not credited from \
+                     account {}.",
+                    payer_account
+                );
+                false
+            }
```

Comment:
Well, not getting a delta isn't an error condition. It just means that the value in the key wasn't written to, so it doesn't show up in the writelog. Failure to read the backing store is an error, but we expect that here since if that happens, the node is in big trouble.

Reply:
I guess it's not expected though. I can add it.